.github/workflows/pull-request.yml

```diff
@@ -16,3 +16,10 @@ jobs:
     uses: firebolt-db/firebolt-python-sdk/.github/workflows/code-check.yml@main
   unit-tests:
     uses: firebolt-db/firebolt-python-sdk/.github/workflows/unit-tests.yml@main
+  security-scan:
+    needs: [unit-tests]
+    uses: firebolt-db/firebolt-python-sdk/.github/workflows/security-scan.yml@main
```

Comment:
you can use `security-scan.yml@<your_branch_name>` to test

Reply:
yes, did it for testing purposes in a separate branch, now want to push it to main